docs/tests.md

```diff
@@ -20,3 +20,9 @@ The following tests are **optional**. Some of them are run by the CI but it is o
 - Use mypy nightly to validate the annotations: `poe mypy --mypy_nightly`
 - Use pyright in full strict mode: `poe pyright_strict`
 - Run stubtest to compare the installed pandas-stubs against pandas (this will fail): `poe stubtest`. If you have created an allowlist to ignore certain errors: `poe stubtest path_to_the_allow_list`
+
+Among the tests above, the following can be run directly during a PR by commenting in the discussion.
+
+- Run pytest against pandas nightly by commenting `\pandas_nightly`
+- Use mypy nightly to validate the annotations by commenting `\mypy_nightly`
+- Use pyright in full strict mode by commenting `\pyright_strict`
```

Comment:
Shouldn't these be forward slashes (e.g., `/pyright_string` rather than `\pyright_strict` )?

Reply:
Forward slashes, my mistake.